lib/timescaledb/migrations/v01.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.Migrations.V01 do
+  @moduledoc false
+
+  use Ecto.Migration
+
+  @spec up() :: any()
+  def up() do
+    create table(:peers_metrics, primary_key: {:id, :id, autogenerate: true}) do
```

Comment:
This is not a valid value for `:primary_key`
```suggestion
    create table(:peers_metrics) do
```
See: https://hexdocs.pm/ecto_sql/Ecto.Migration.html#table/2